packages/shared/components/popups/SendConfirmationPopup.svelte

```diff
@@ -51,8 +51,8 @@
     $: isInternal = recipient.type === 'account'
     $: isNativeToken = asset?.id !== $selectedAccountAssets?.baseCoin?.id
 
-    $: expirationDate, void _prepareOutput()
-    $: expirationDate, (error = null)
+    $: expirationDate, giftStorageDeposit, void _prepareOutput()
+    $: expirationDate, giftStorageDeposit, (error = null)
```

Comment:
Could create one reactive statement out of two:
```suggestion
    $: expirationDate, giftStorageDeposit, refreshSendConfirmationState()

    function refreshSendConfirmationState(): void {
        error = null
        void _prepareOutput()
    }
```